tests/plugins/fakeobjects/manual/pastefakeelement.md

```diff
@@ -0,0 +1,33 @@
+@bender-tags: 4.8.0, bug, clipboard, 638
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, iframe, clipboard, sourcearea, link
+
+**Scenario 1:**
+
+1. Open this page in new tab.
+2. Click `Source` in editor and paste:
+`<p>Lorem ipsum</p><p><iframe frameborder="0" height="100" src="https://example.com" width="100"></iframe></p><p>cke-real-element-type</p>`
```

Comment:
This step could be omitted and instead this HTML could be already inserted into the editor.